src/util.test.ts

```diff
@@ -24,22 +24,23 @@ describe("Util", (): void => {
 
     const mockedExec = async (
       error: Error | null,
-      stdout: string | null = null,
-      stderr: string | null = null
+      stdout: string = "",
+      stderr: string = ""
     ): Promise<void> => {
       child_process.exec.mockImplementationOnce(
         (_command: any, _options: any, callback: any): any => {
           callback(error, { stdout, stderr });
         }
       );
-      await util.execBashCommand(COMMAND);
+      const output = await util.execBashCommand(COMMAND);
 
       expect(core.info).nthCalledWith<[string]>(1, COMMAND);
       expect(child_process.exec).lastCalledWith(
         COMMAND,
         { shell: "/usr/bin/bash" },
         expect.anything()
       );
+      expect(output).toBe(stdout);
```

Comment:
When `error` is not `null`, we expect `output` to be `""`. The existing test enforces this expectation, but does it in something of a cryptic way spanning two levels of abstraction that will be more error-prone in the future.